charts/nginx-gateway-fabric/templates/deployment.yaml

```diff
@@ -90,6 +90,10 @@ spec:
         lifecycle:
         {{- toYaml .Values.nginxGateway.lifecycle | nindent 10 }}
         {{- end }}
+        {{- if .Values.nginxGateway.resources }}
+        resources:
```

Comment:
Can you add this field to the values.yaml file?

Reply:
For sure, what kind of defaults values would you like it to have?  
Reasoning from my side was to not set it, just so it follows the current default behavior.  

We're currently pre-production with our setup using this gateway, so we have no "real" life data on how much CPU and/or memory we expect it to be using.

Reply:
nvm, didn't notice that most values like those have a default empty struct, added that one :)